tock/tock/tests/js/integration/timecard.test.js

```diff
@@ -46,6 +46,25 @@ describe("Timecard", () => {
       expect(checked).toEqual(false);
     });
 
+    test('checks if there is a checkbox image when image is checked', async () => {
+      const entries = await page.$$(".entry");
+      const length = entries.length;
+      // Find and check the last delete input on the page
+      const last_entry_idx = length - 1
+      const _del = "#id_timecardobjects-" + last_entry_idx + "-DELETE";
+      await page.$$eval(_del, checks => checks.forEach(c => c.checked = true));
+
+      // add new entry
+      await page.click(".add-timecard-entry");
+
+      const _new_delete_input = await page.$("#id_timecardobjects-" + length + "-DELETE");
```

Comment:
(Nitpick) This (and below) would be a great use of template strings with backticks! 

```js
page.$(`#id_timecardobjects-${length}-DELETE`);
```